go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/aperture v0.1.6-beta
-	github.com/lightninglabs/lndclient v0.14.0-8
+	github.com/lightninglabs/lndclient v0.14.2-3
```

Comment:
We need to update `LoopMinRequiredLndVersion` since we're using a new api param. 

We could remove it completely so that we just use the lndclient default every time we bump. I'm somewhat in favor of this, since it's easy to forget to bump, but the downside is more upgrade pressure for clients.